client/src/workspace/components/SystemStatusModal/SystemStatusModal.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import { Modal, Badge, Spin, Alert} from 'antd';
+import { SystemQueueTable } from './SystemQueueTable';
+import { useSystemOverview } from '../../../hooks/system-status/useSystemOverview';
+import styles from './SystemStatusModal.module.css';
+
+interface SystemStatusModalProps {
+  isModalVisible: boolean;
+  onClose: () => void;
+}
+
+export const SystemStatusModal: React.FC<SystemStatusModalProps> = ({
+  isModalVisible,
+  onClose,
+}) => {
+  const [activeSystem, setActiveSystem] = useState('Stampede3');
+
+  const { data: systems, isLoading, isError, error } = useSystemOverview();
+
+  const selectedSystem = systems?.find(
+    (sys) => sys.display_name.toLowerCase() === activeSystem.toLowerCase()
+  );
+
+  //setting color for status of system
+  const getStatusColor = (system: typeof selectedSystem) => {
+    if (system?.is_operational) return 'success';
+    if (system?.in_maintenance) return 'warning';
+    return 'error';
+  };
+
+  return (
+    <Modal
+      title="System Status"
+      open={isModalVisible}
+      onCancel={onClose}
+      footer={null}
+      width={500}
+    >
+      <div className={styles.modal}>
+        <div className={styles.tabs}>
+          {['frontera', 'lonestar6', 'Stampede3'].map((hostname) => (
```

Comment:
Instead of hardcoding these, let's try doing it dynamically with the response data from tap. e.g. from https://tap.tacc.utexas.edu/status/, we can iterate over each key in the response dict, and assume it is of the form of a system object. We can further ensure good data by filtering the response json by checking that `sys?.system_type === 'COMPUTE'`